request.go

```diff
@@ -716,14 +758,37 @@ func (req *SelectRequest) Key(key interface{}) *SelectRequest {
 	return req
 }
 
+// FetchPos determines whether to fetch positions of tuples. A position
+// descriptor will be saved in Response.Pos value.
+//
+// Requires Tarantool >= 2.11.
+//
+// Note: default value is false.
```

Comment:
Shouldn't we also have "supported since 1.11.0" here?

Reply:
Added.